src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,249 @@
+<script lang="ts">
+  import type { QuizData, TeamMatch } from '$lib/constants/quiz';
+  import LeftArrow from '$lib/components/icons/left-arrow.svelte';
+  import RightArrow from '$lib/components/icons/right-arrow.svelte';
+  import MoreInfo from './more-info.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: TeamMatch[] = [];
+  $: answeredAllQuestions =
+    response.length === data.questions.length && !response.includes(undefined);
```

Comment:
We can build a reactive statement that automatically stores the user's responses in [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
```suggestion
  let answeredAllQuestions = false;
  $: {
    answeredAllQuestions =
      responses.length === data.questions.length && !responses.includes(undefined);
    // Updates local storage whenever `responses` changes.
    quizStorage.setResponses(responses);
  }
```

Reply:
done